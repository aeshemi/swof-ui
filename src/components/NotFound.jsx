import React from 'react';
import { Jumbotron } from 'react-bootstrap';

const NotFound = () => (
  <Jumbotron>
    <h1>Page Not Found!</h1>
  </Jumbotron>
);

export default NotFound;
