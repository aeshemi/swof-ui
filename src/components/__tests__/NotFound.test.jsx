import NotFound from '../NotFound';

it('NotFound renders correctly', () => {
  const wrapper = shallow(<NotFound />);
  expect(wrapper).toMatchSnapshot();
});
