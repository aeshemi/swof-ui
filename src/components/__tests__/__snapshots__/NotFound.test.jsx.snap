// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
<Jumbotron
  bsClass="jumbotron"
  componentClass="div"
>
  <h1>
    Page Not Found!
  </h1>
</Jumbotron>
`;
